01-bash-scripting/02_connectivity_list_of_websites_1.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+websites=(
+    "https://www.klix.ba"
+    "https://www.google.com"
+    "https://www.facebook.com"
+    "https://www.sportsport.ba"
+    "https://www.laznastranica.ba"
+)
+
+for website in "${websites[@]}"; do
+    if curl --output /dev/null --silent --head --fail "$website"; then
```

Comment:
There is no need for `--silent` and `--output /dev/null` together. `--silent` is enough for this use case.